.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,46 @@
+---
+name: New Release
+about: Propose a new release
+title: Release v0.x.0
+labels: ''
+assignees: ''
+
+---
+
+## Release Checklist
+- [ ] For major or minor releases (v$MAJ.$MIN.0), create a new release branch.
+  - [ ] A maintainer creates a vanilla release branch with
+        `git branch release-$MAJ.$MIN main`
+  - [ ] A maintainer pushes the new release branch with
+        `git push release-$MAJ.$MIN` (e.g., release-0.1)
+- [ ] Update things like README, deployment templates, docs, configuration, test/e2e flags.
+      Submit a PR against the release branch.
+- [ ] A maintainer [prepares a draft release](https://github.com/kubernetes-sigs/gateway-api-inference-extension/releases)
+  - [ ] Write the change log into the draft release.
+  - [ ] Run
+      `make artifacts`
+      to generate the artifacts and upload the files in the `artifacts` folder to the draft release.
```

Comment:
"upload the files in the `artifacts` folder to the draft release." is "Attach binaries by dropping them here or selecting them." in the release webpage, correct?

Reply:
It is the yamls under the artifacts folder that got created by `make artifacts`, github automatically creates and attaches a compressed archive of the code.